mtest/mscore/palette/tst_palette.cpp

```diff
@@ -64,6 +64,13 @@ void TestPaletteModel::testDuplicateItemNames()
       qDebug("In %s workspace", qPrintable(workspaceName));
       for (auto name = paletteItemNames.begin(); name != paletteItemNames.end(); ++name) {
             if (name.value().size() != 1) {
+                  // Exceptions - allowed duplicates
+                  if (name.key().endsWith(" repeat sign") // repeat barlines in "Barlines" and "Repeats & Jumps" palette
+                      || name.key() == "Open" // articulations in "Articulations" and channel switch text in "Text" palette
+                      || name.key() == "Line" // bracket type in "Brackets" and line type in "Lines" palette
+                      || name.key().startsWith("Add parentheses to ") // "Noteheads" and "Accidentals" palette
+                      )
+                        continue;
```

Comment:
Instead of `if(cond) continue;` we could do `if(!cond) duplicates = true;` That would make the debug statement below print out the duplicates if you run the test. However, IIRC, Travis doesn't show debug output if the tests pass (and no one would check it anyway even if it did), so I don't know how we could show the warnings.

Reply:
If the warnings don't show, they won't be worth being added